### PR TITLE
Community common scripts 2-12

### PR DESCRIPTION
**Pull request by RotemAmit**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Moved first 11 scripts to CommunityCommonScripts

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommunityCommonScripts/.pack-ignore [hunk 1]
```diff
@@ -0,0 +1,56 @@
+[file:pack_metadata.json]
+ignore=PA125
+
+[file:Scripts/BatchData/BatchData.yml]
+ignore=BA114
```
Comment (Shellyber) [inline Packs/CommunityCommonScripts/.pack-ignore:5]: remove those ignores (BA114) - no need

File: Packs/AttachmentsPreProcessing/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+##### CreateFileFromPathObject
+- Deprecated. Use CommunityCommonScripts pack instead.
```
Comment (Shellyber) [inline Packs/AttachmentsPreProcessing/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Deprecated. Use the Community Common Scripts pack instead.
```
Fix everywhere accordingly

File: Packs/CommunityCommonScripts/pack_metadata.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Community Common Scripts",
+    "description": "A pack that contains community scripts",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "",
+    "url": "https://live.paloaltonetworks.com/t5/cortex-xsoar-discussions/bd-p/Cortex_XSOAR_Discussions",
```
Comment (Shellyber) [inline Packs/CommunityCommonScripts/pack_metadata.json:7]: Is this the right URL?

File: Packs/CreateEDLInstance/ReleaseNotes/1_0_2.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### CreateEDLInstance
+- Moved to the *CommunityCommonScripts* pack.
```
Comment (Shellyber) [inline Packs/CreateEDLInstance/ReleaseNotes/1_0_2.md:4]:
```suggestion
- Deprecated. please use Community Common Scripts instead.
```

File: Packs/DynamicSectionReports/ReleaseNotes/1_0_2.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+##### DisplayIndicatorReputationContent
+- Moved to the *CommunityCommonScripts* pack.
```
Comment (Shellyber) [inline Packs/DynamicSectionReports/ReleaseNotes/1_0_2.md:4]: Fix this

File: Packs/DynamicSectionReports/ReleaseNotes/1_0_2.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+
+##### DisplayTaggedWarroomEntries
+- Moved to the *CommunityCommonScripts* pack.
```
Comment (Shellyber) [inline Packs/DynamicSectionReports/ReleaseNotes/1_0_2.md:7]: same

**Review by Shellyber — COMMENTED** [review]
Nice

**Review by Shellyber — APPROVED** [review]
Nice!

**Comment by RotemAmit** [comment]
The failing validations are all BA114. It happens because we moved the scripts from one pack to another.

**Pull request merged**